_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -997,10 +1042,10 @@ Set the variables for the `account-baseline-root` module in this environment in
 [source,hcl]
 ----
 locals {
-  aws_region = "us-east-1"
+  aws_region = local.aws_region
 
   accounts = {
-    root = "216044045972"
+    root = "<ROOT_ACCOUNT_ID>"
```

Comment:
So should this root account be in the `accounts.hcl`?

Reply:
I think we can leave it like this, since we ask them to create the root account manually first.